api/src/types.rs

```diff
@@ -138,8 +138,11 @@ impl EncryptedBody {
 #[derive(Serialize, Deserialize, Debug, Clone)]
 #[serde(untagged)]
 pub enum RpcId {
+	/// None
```

Comment:
Do you know more details after this information? Can you add to the text?

Reply:
This enum is kind of obscure because it seems to only keep IP's, so when it doesn't have any IP I don't know what to put in the documentation.